shell/src/main/java/org/apache/accumulo/shell/commands/CreateTableCommand.java

```diff
@@ -86,14 +86,14 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
     // and then the addSplits method was called. Starting with 2.0, the splits will be
     // stored on the file system and created before the table is brought online.
     if (cl.hasOption(createTableOptSplit.getOpt())) {
-      ntc = ntc.withSplits(new TreeSet<>(
+      ntc.withSplits(new TreeSet<>(
           ShellUtil.scanFile(cl.getOptionValue(createTableOptSplit.getOpt()), decode)));
     } else if (cl.hasOption(createTableOptCopySplits.getOpt())) {
       final String oldTable = cl.getOptionValue(createTableOptCopySplits.getOpt());
       if (!shellState.getAccumuloClient().tableOperations().exists(oldTable)) {
         throw new TableNotFoundException(null, oldTable, null);
       }
-      ntc = ntc.withSplits(
+      ntc.withSplits(
```

Comment:
The change is correct, but I lament the fact that we chose these method names poorly. I think "with" implies that the returned value is "with" the changes, but the original is "without", unless you have something to indicate otherwise. But, the javadocs do explicitly imply the object is being mutated, since it says `@return this` (at least on some methods... it should probably do that on more of them).

Reply:
Updated javadoc for NTC in a13c742ef7